### ISSUE TITLE
Weather_code variable crashes docker container

### ISSUE THREAD

**Issue by hundredfire** [thelande/openmeteo_exporter#1]
Hello, thank you for your work. As openweathermap's free API is bound to be deprecated, I expect your project to gain popularity. 
I managed to deploy a docker container with your exporter with different variables, but noticed that the weather_code variable crashed it. 
Here is my config.yaml:
```
---
locations:
  - name: Nice
    latitude: 43.7102 
    longitude: 7.2620
    timezone: Europe/Paris
    weather:
      variables:
        - temperature_2m
        - relative_humidity_2m
        - precipitation_probability
        - weather_code
    air_quality:
      variables:
        - pm2_5
        - european_aqi
``` 

And here's the relevant part of the log:

```
ts=2024-04-12T23:55:49.460Z caller=main.go:62 level=info msg="Starting openmeteo_exporter" version="(version=1.0.1, branch=HEAD, revision=6d9a9764484333907cae472b1e0a05056980786f)"
ts=2024-04-12T23:55:49.460Z caller=main.go:63 level=info msg="Build context" build_context="(go=go1.21.7, platform=linux/amd64, user=root@buildkitsandbox, date=20240303-18:25:13, tags=netgo osusergo static_build)"
ts=2024-04-12T23:55:49.460Z caller=config.go:80 level=info msg="Loaded configuration file" path=/config.yaml locations=1
ts=2024-04-12T23:55:49.462Z caller=tls_config.go:274 level=info msg="Listening on" address=[::]:9812
ts=2024-04-12T23:55:49.462Z caller=tls_config.go:277 level=info msg="TLS is disabled." http2=false address=[::]:9812
panic: "openmeteo_weather_weather_code_wmo code" is not a valid metric name
goroutine 14 [running]:
github.com/prometheus/client_golang/prometheus.MustNewConstMetric(...)
	/go/pkg/mod/github.com/prometheus/client_golang@v1.17.0/prometheus/value.go:129
main.WeatherCollector.Collect({0xe5a6e0?, 0xc000085ea8?}, 0xc000085e88?)
	/src/collector_weather.go:67 +0x799
main.OpenMeteoCollector.Collect({0xe5a6e0?, {0xc000023e00?, 0x0?, 0x0?}}, 0x0?)
	/src/collector.go:73 +0x28e
github.com/prometheus/client_golang/prometheus.(*Registry).Gather.func1()
	/go/pkg/mod/github.com/prometheus/client_golang@v1.17.0/prometheus/registry.go:455 +0x102
created by github.com/prometheus/client_golang/prometheus.(*Registry).Gather in goroutine 12
	/go/pkg/mod/github.com/prometheus/client_golang@v1.17.0/prometheus/registry.go:466 +0x568

``` 

Also, I'd like to know if it would be possible to add a variable for Celsius degrees? Thank you for your work again.

**Comment by thelande** [comment]
Hi there! Thanks for letting me know about the issue. I think I already see what the problem is so I'll try to get it fixed soon. In the meantime, I updated the README with information about how to configure the units. Support was already there but I just failed to note how to use it.

https://github.com/thelande/openmeteo_exporter?tab=readme-ov-file#unit-configuration

**Comment by thelande** [comment]
Issue should be fixed in v1.0.2. Thanks again for reporting this.

**Comment by hundredfire** [comment]
Impressive work. Thank you very much. Cheers
